pkgs/development/libraries/xxHash/default.nix

```diff
@@ -11,21 +17,16 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-2WoYCO6QRHWrbGP2mK04/sLNTyQLOuL3urVktilAwMA=";
   };
 
-  # Upstream Makefile does not anticipate that user may not want to
-  # build .so library.
-  postPatch = lib.optionalString stdenv.hostPlatform.isStatic ''
-    sed -i 's/lib: libxxhash.a libxxhash/lib: libxxhash.a/' Makefile
-    sed -i '/LIBXXH) $(DESTDIR/ d' Makefile
-  '';
+  patches = [ ./man_location.patch ];
```

Comment:
Could you link to an upstream issue/PR addressing this?

Reply:
I fixed it to use the upstream patch, and linked to the issue